vuepress/docs/v7.1/tutorials/create/ms/add-api-claim.md

```diff
@@ -3,6 +3,8 @@ Micro frontends typically require data or logic provided by microservices. The c
 
 :::tip Note
 API claims in Entando can be used for microservices within the same bundle or for microservices within another bundle in the same namespace. This tutorial demonstrates the first case. 
+
+Entando recommends creating REST APIs in microservices instead of extending the App Engine as previously done.
```

Comment:
super minor - can remove "as previously done" because why draw attention to a method that is no longer recommended/supported... but keeping "instead of extending the app engine" is good - someone might decide that's another way to do it that's just as acceptable or easier, etc

Reply:
I agree with you about not wanting to draw attention to it and I waffled both ways. In the end, I think it gives it context that is helpful. I do like how it sounds without 'as previously done' but ending it like that seemed to infer they are both ok options but one is just more recommended. This way, people see it is an evolution